server/src/main/java/org/elasticsearch/index/search/QueryStringQueryParser.java

```diff
@@ -490,7 +489,8 @@ private Query getFuzzyQuerySingle(String field, String termStr, int minSimilarit
                 getFuzzyPrefixLength(),
                 fuzzyMaxExpansions,
                 fuzzyTranspositions,
-                context
+                context,
+                null
```

Comment:
This should probably pass `this.fuzzyRewriteMethod` shouldn't it? But I guess that's a behaviour change - maybe add a TODO and we can call it out separately as a bugfix?

Reply:
OK, looking at `QueryParserBase#getFuzzyQuery` (the thing this class overrides), Lucene doesn't set the `fuzzyRewriteMethod` there either. 

Do we want to do that here? I am honestly not sure.